exercises/basiccalcobject.js

```diff
@@ -3,24 +3,59 @@ class Calculator{
 	constructor(){
 		//takes in nothing
 		//make storage for the operator and the numbers
+		this.operator = [];
+		this.numbers = [];
 	}
-	loadOperator(  ){
+	loadOperator( operator ){
 		//adds the operator to the next calculation
 		//takes in the operator
+		if(operator === '+' || operator === '-' || operator === '*' || operator === '/'){
```

Comment:
could have made  method to say whether the op is a valid one.

also could have put the values into an array or string and done indexOf to find it.
Or could have put them into an object to find it, with all the values being true, like 
```
var validOps = { '+': true, '*': true, '-': true } 
```
or even made then translate to a value
``` 
var validOps = { '+': '+', '*': '*', 'x': '*', 'X': '*', '-':'-'}
```
or used a switch with no breaks.